features/map.go

```diff
@@ -27,14 +27,15 @@ type mapCache struct {
 
 func createMapTypeAttr(mt ebpf.MapType) *sys.MapCreateAttr {
 	var (
-		keySize        uint32 = 4
```

Comment:
Somewhat OT: just filling in `MapCreateAttr` would avoid duplicating all field names.

Reply:
Sure, will pick this up.